Fix: Add Search button in Lobbies, fix Copy ID error, update README (…

JessTello opened the pull request:
- Added visible "Search" button in Lobbies scene
- Fixed Copy ID error
- Add usage instructions for Disabled Anti-Cheat toggle in README

JessTello committed:
Fix: Add Search button in Lobbies, fix Copy ID error, update README (EOSU-655)

- Added visible "Search" button in Lobbies scene
- Fixed Copy ID error
- Add usage instructions for Disabled Anti-Cheat toggle in README

matt-clarke commented on Assets/Scripts/StandardSamples/UI/Lobbies/UILobbiesMenu.cs:
```diff
@@ -91,11 +88,11 @@ public class UILobbiesMenu : SampleMenuWithFriends
+                const bool ONANDROIDPLATFORM = true;
+        #else
+                const bool ONANDROIDPLATFORM = false;
+        #endif
```
```suggestion
#if UNITY_ANDROID && !UNITY_EDITOR //TODO: this should be in a centralized class to reduce clutter, and like an enum if other platforms are to be included
        const bool ONANDROIDPLATFORM = true;
#else
        const bool ONANDROIDPLATFORM = false;
#endif
```

matt-clarke commented on Assets/Scripts/StandardSamples/UI/Lobbies/UILobbiesMenu.cs:
```diff
@@ -76,10 +77,6 @@ public class UILobbiesMenu : SampleMenuWithFriends
-        
-        
-
         // UI Cache
```
```suggestion

        // UI Cache
```

matt-clarke commented on Assets/Scripts/StandardSamples/UI/Lobbies/UILobbiesMenu.cs:
```diff
@@ -796,5 +794,29 @@ public void ToggleDeafenSelf()
+            }
+            else if (!string.IsNullOrEmpty(bucket))
+            {
+                LobbyManager.SearchByAttribute("bucket", bucket, UIUpateSearchResults);
```
Can you add consts for these attribute keys to LobbyManager and use them everywhere?

billfr-ApexSystems commented:
Agree with Matt's suggested updates.  @JessTello Let's review for other const clean-up we can accomplish in the module as well.

JessTello committed:
Refactor: Formatting adjustments; Introduce consts for LobbyManager

- Add consts for attribute keys in LobbyManager and apply them globally
- Fix indentation and spacing

matt-clarke commented on Assets/Scripts/StandardSamples/UI/Lobbies/UILobbiesMenu.cs:
```diff
@@ -88,11 +89,11 @@ public class UILobbiesMenu : SampleMenuWithFriends
-        #else
-                const bool ONANDROIDPLATFORM = false;
-        #endif
+#if UNITY_ANDROID && !UNITY_EDITOR //TODO: this should be in a centralized class to reduce clutter, and like an enum if other platforms are to be included
+    const bool ONANDROIDPLATFORM = true;
+#else
+    const bool ONANDROIDPLATFORM = false;
+#endif
```
```suggestion
#if UNITY_ANDROID && !UNITY_EDITOR //TODO: this should be in a centralized class to reduce clutter, and like an enum if other platforms are to be included
        const bool ONANDROIDPLATFORM = true;
#else
        const bool ONANDROIDPLATFORM = false;
#endif
```

matt-clarke committed:
Fix indent

billfr-ApexSystems reviewed (approved):
Approved with Matt's changes requested
